### DISCUSSION TITLE
Cart API Suggestion

### DISCUSSION THREAD

**Discussion by osseonews** [swellstores/community#31] [Ideas]
The call to add an item to a cart with JS, `swell.cart.addItem`, requires a productId and variantId from Swell. The problem is that these IDs are unique to Swell only, and if you are in development, you will need to maintain 2 lists of ID's, one for development and another for live, as the ID's will different on each Swell instance for the same exact item. This makes development difficult for various reasons. It would better if the Cart used a SKU as the unique code for all cart functionality, instead of a Swell generated Id. The SKU is unique to the company, as opposed to Swell, and will make it much easier to develop as there will be a unique sku code that will be the same across any environment.

**Comment by ericingram** [comment]
That's a great idea. We'll add it to a list of improvements!
